wapitiCore/attack/mod_csp.py

```diff
@@ -65,21 +114,14 @@ async def attack(self, request: Request, response: Optional[Response] = None):
             )
         else:
             csp_dict = csp_header_to_dict(response.headers["Content-Security-Policy"])
-
+            info = ""
             for policy_name in CSP_CHECK_LISTS:
-                result = check_policy_values(policy_name, csp_dict)
-
-                if result <= 0:
-                    if result == -1:
-                        info = MSG_CSP_MISSING.format(policy_name)
-                    else:  # result == 0
-                        info = MSG_CSP_UNSAFE.format(policy_name)
-
-                    log_red(info)
-                    await self.add_vuln_low(
-                        category=NAME,
-                        request=request_to_root,
-                        info=info,
-                        wstg=WSTG_CODE,
-                        response=response
-                    )
+                info += check_policy(policy_name, csp_dict)
```

Comment:
Currently, we check if the most important rules are well set
Shouldn't we also check for mistyped/non-existant CSP by being more strict ? (notify the user of weird CSP like someone who would have typed ``deflaut-src``)

Reply:
At present, we only check four directives, and it is difficult to check this case out.
The current version of the module will state that "default-src" is not in the CSP for "deflaut-src" incorrectly typed, for example.
but adding this verification later could be interesting